### ISSUE TITLE
InputNumber: change ajax behavior doesn't fire if you click on draggable components

### ISSUE THREAD

**Issue by szati64** [primefaces/primefaces#8787]
### Describe the bug

Hi!
We have encountered a bug when you try to modify the value of an inputNumber which has a change ajax behavior attached to it then the event doesn't always fire. In normal cases if you click outside of the input field the component formats the number you input there and then fires the change ajax. There is a special case however if you click outside of the input field but the object you clicked on is a draggable one. In this case the formatting still executes but the change ajax is not called. Same thing happens if you try to move the draggable object instead of click on it.

### Reproducer

https://github.com/szati64/primefaces-issue-8787-reproducer

I already found two cases to reproduces this issue.

**Case one**: In the first one I have a p:dragabble object. If you modify the value of input1 inputNumber and then click on the dragabble examplePanel or try to move this object you can see the lastUpdate label will not update. Also if you try to close the panel instead of moving it the change ajax doesn't execute.

case_one.xhtml:

```
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:h="http://java.sun.com/jsf/html"
	xmlns:f="http://java.sun.com/jsf/core"
	xmlns:p="http://primefaces.org/ui">

<h:head>
</h:head>

<h:body>

	<h:form>

		<p:outputPanel>
			<p:outputLabel for="input1" value="Example input:" />
			<p:inputNumber id="input1" value="#{caseOneController.input1}">
				<p:ajax event="change"
					listener="#{caseOneController.input1Changed()}" update="@form" />
			</p:inputNumber>
			<p:outputLabel value="#{caseOneController.lastUpdate}"
				rendered="#{not empty caseOneController.lastUpdate}" />
		</p:outputPanel>

	</h:form>

	<p:panel id="examplePanel" header="Title" widgetVar="ExamplePanel"
		toggleable="true" style="max-width: 200px">
			Text
	</p:panel>

	<p:draggable for="examplePanel" />

</h:body>
</html>
```

CaseOneController.java:

```
@Named
@ViewScoped
public class CaseOneController implements Serializable {

    private BigDecimal input1;
    private LocalDateTime lastUpdate;

    public void input1Changed() {
        lastUpdate = LocalDateTime.now();
    }

    public BigDecimal getInput1() {
        return input1;
    }

    public void setInput1(BigDecimal input1) {
        this.input1 = input1;
    }

    public LocalDateTime getLastUpdate() {
        return lastUpdate;
    }

    public void setLastUpdate(LocalDateTime lastUpdate) {
        this.lastUpdate = lastUpdate;
    }

}
```

**Case two**: In the 2nd case I put all the fields into a dialog which you can reposition by click on the header. If you move the dialog after you modified the input1 inputNumber field you can see the change ajax doesn't fire and the lastUpdate label will not update either.

case_two.xhtml:

```
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:h="http://java.sun.com/jsf/html"
	xmlns:f="http://java.sun.com/jsf/core"
	xmlns:p="http://primefaces.org/ui">

<h:head>
</h:head>

<h:body>

	<p:commandButton value="Open dialog" onclick="PF('exampleDlg').show()" />

	<p:dialog header="Header" widgetVar="exampleDlg" minHeight="40"
		width="350" modal="true">
		<h:form>
			<p:outputPanel>
				<p:outputLabel for="input1" value="Example input:" />
				<p:inputNumber id="input1" value="#{caseTwoController.input1}">
					<p:ajax event="change"
						listener="#{caseTwoController.input1Changed()}" update="@form" />
				</p:inputNumber>
				<p:outputLabel value="#{caseTwoController.lastUpdate}"
					rendered="#{not empty caseTwoController.lastUpdate}" />
			</p:outputPanel>
		</h:form>
	</p:dialog>

</h:body>
</html>
```

CaseTwoController.java:

```
@Named
@ViewScoped
public class CaseTwoController implements Serializable {

    private BigDecimal input1;
    private LocalDateTime lastUpdate;

    public void input1Changed() {
        lastUpdate = LocalDateTime.now();
    }

    public BigDecimal getInput1() {
        return input1;
    }

    public void setInput1(BigDecimal input1) {
        this.input1 = input1;
    }

    public LocalDateTime getLastUpdate() {
        return lastUpdate;
    }

    public void setLastUpdate(LocalDateTime lastUpdate) {
        this.lastUpdate = lastUpdate;
    }

}
```

### Expected behavior

If you click anywhere outside of an p:inputNumber field after you modfied its value the value should format and the change ajax behavior should fire.

### PrimeFaces version

10.0, 11.0

### JSF implementation

Mojarra

### JSF version

2.3

**Comment by melloware** [comment]
Please provide an executable example using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Comment by szati64** [comment]
Hi! I have made an executable example based on PrimeFaces Test project and I added the two reproducer cases I mentioned in the issue: https://github.com/szati64/primefaces-issue-8787-reproducer

**Comment by melloware** [comment]
Good reproducer.  This is a tricky one.  I don't quite see it but it looks like when you click on the draggable its somehow stopping propagation of the change event.  Because this seems to work if you click on anything else other than a draggable.  Very strange.

**Comment by melloware** [comment]
I just tested this with PrimeFaces 13.0.0 and it is fixed. I can only assume its related to AutoNumeric or some other fix.
